sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsLiveTests.cs

```diff
@@ -0,0 +1,700 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Errors;
+using Azure.Messaging.EventHubs.Metadata;
+using Azure.Messaging.EventHubs.Tests.Infrastructure;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of live tests for the <see cref="TrackOne.EventHubsDiagnosticSource" />
+    ///   class.
+    /// </summary>
+    ///
+    /// <remarks>
+    ///   These tests have a depenency on live Azure services and may
+    ///   incur costs for the associated Azure subscription.
+    ///
+    ///   Every send or receive call will trigger diagnostics events as
+    ///   long as they are being listened to, making it possible for other
+    ///   tests to interfere with these. For this reason, these tests are
+    ///   marked as non-parallelizable.
+    ///
+    /// </remarks>
+    ///
+    [TestFixture]
+    [NonParallelizable]
+    [Category(TestCategory.Live)]
+    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
+    public class DiagnosticsLiveTests
```

Comment:
At some point, we'll want to refactor these a bit, since the track one tests aren't structured around the "test one and only one thing" principle.  For now, I think we're good, but I'd appreciate if you could open an issue and assign it to me for tracking.

Reply:
Done: #6970.